docs/tutorial/part-three/index.md

```diff
@@ -232,8 +232,8 @@ const ListLink = props => (
 // highlight-end
 
 export default ({ children }) => (
+  {/* highlight-start */}
   <div style={{ margin: `0 auto`, maxWidth: 650, padding: `1.25rem 1rem` }}>
```

Comment:
Maybe we should update styling here to be same as in previous code snippet instead?